crates/autopilot/src/database/competition.rs

```diff
@@ -42,6 +43,11 @@ pub struct Competition {
     /// chain before this block height.
     pub block_deadline: u64,
     pub order_executions: Vec<OrderExecution>,
+    pub competition_simulation_block: u64,
+    /// Winner settlement call data
+    pub call_data: Vec<u8>,
+    /// Uninternalized winner settlement call data
+    pub uninternalized_call_data: Vec<u8>,
```

Comment:
Could probably use [ByteArray](https://github.com/cowprotocol/services/pull/1051/files) to get nicer Debug printing for the calldata bytes.

Reply:
But ByteArray requires size to be known at compile time?

Reply:
Ahh, right. Alternatively you should be able to use [this](https://github.com/cowprotocol/services/pull/836/files).

Reply:
Nice, added.